### ISSUE TITLE
Add Karmada starting examples through binary or source code

### ISSUE THREAD

**Issue by JadeFlute0127** [karmada-io/playground#7]
What would you like to be added:
Add Karmada starting examples with [killercoda](https://killercoda.com/), including a binary installation example and script installation example, both containing installation and deploying workload to multi-clusters steps.

Tasks & deadline:

1. Implement binary installation example, with deploying workload to multi-clusters steps.
- [ ] Implement CLI installation example
- [ ] Using CLI to implement the steps of deploying workloads to multi-clusters
2. Implement script installation example, with deploying workload to multi-clusters steps.
- [ ] Implement script installation example
- [ ] Using scripts to implement the steps of deploying workloads to multi-clusters

**Comment by RainbowMango** [comment]
Please @jwcesign confirm that's the plan.

**Comment by jwcesign** [comment]
Hi, @JadeFlute0127

We already have the demo of ` Implement CLI installation example`, what's new?

**Comment by JadeFlute0127** [comment]
> Hi, @JadeFlute0127
> 
> We already have the demo of ` Implement CLI installation example`, what's new?

I have encountered some users in the community who may be limited by the network environment and unable to easily use CLI tools. Offline installation from source code is more suitable for their scenarios. However, during the deployment process, insufficient knowledge of the background of K8S and Docker resulted in encountering some unpredictable difficulties. This scenario aims to provide a tutorial for these people to build karmada from the source code.

**Comment by jwcesign** [comment]
It makes sense.

But I think we have higher priority things to finish.
- Provide an architecture diagram illustrating how the deployment is propagated in existing scenarios.
- Enhance the speed of environment setup, for example by utilizing kind clusters creation way. Alternatively, if it's not possible to expedite the process, provide more user-friendly information instead of making users wait for 90 seconds.

**Comment by jwcesign** [comment]
/cc @JadeFlute0127 

What do you think?

**Comment by JadeFlute0127** [comment]
> It makes sense.
I agree with you.As you mentioned, these are indeed more urgent matters, and I will seek solutions in parallel, including but not limited to seeking other more suitable platforms.

**Comment by jwcesign** [comment]
Thanks! @JadeFlute0127

**Comment by RainbowMango** [comment]
Please ignore me, I'm testing the robot.
/joke
